3rd-party/cove/app/UIProgressDialog.cpp

```diff
@@ -43,8 +43,9 @@ UIProgressDialog::UIProgressDialog(const QString& labelText,
 {
 	pDialog.setMinimumDuration(0);
 	pDialog.setWindowModality(Qt::WindowModal);
-	connect(this, SIGNAL(percentageUpdated(int)), &pDialog,
-			SLOT(setValue(int)));
+	connect(this, &UIProgressDialog::percentageUpdated,
+	        &pDialog, &QProgressDialog::setValue,
+	        Qt::QueuedConnection);
```

Comment:
This could be the reason for crashes reported in issue GH-1470. I recall fixing the inter-thread signaling and I'm surprised that it was wrong at this spot.
...
And this is where I broke it: https://github.com/lpechacek/mapper/commit/9bac3fd2519a110a754fbb28cf4cd869d682462b
Funnily, the commit message talks about queued connections. :)

Thanks for catching this!

Reply:
Hm, maybe QueuedConnection shoots back. Now the progress update is put into the main thread's event queue, but running the event loop is no longer triggered...